Stripe/PublicHeaders/STPAPIClient+ApplePay.h

```diff
@@ -45,15 +45,16 @@ NS_ASSUME_NONNULL_BEGIN
                             completion:(STPPaymentMethodCompletionBlock)completion;
 
 /**
- Converts Stripe errors into the appropriate Apple Pay error, for use in `PKPaymentAuthorizationResult`. The error is displayed in the Apple Pay sheet, and the user can try again.
+ Converts Stripe errors into the appropriate Apple Pay error, for use in `PKPaymentAuthorizationResult`.
  
- We can convert billing address related errors into a PKPaymentError that helpfully points to the billing address field in the Apple Pay sheet.
- All other errors map to PKPaymentUnknownError, resulting in a generic error message in the Apple Pay sheet.
+ If the error can be fixed by the customer within the Apple Pay sheet, we return an NSError that can be displayed in the Apple Pay sheet.
```

Comment:
based on changelog wouldn't this result in sheet dismissal?

Reply:
Yep. See the ticket for more context, but I think this is the right UX - there's a limited set of errors that can be displayed / are actionable within the Apple Pay sheet.

Reply:
I'm confused about why this says "can be fixed by the customer within the Apple Pay sheet" if the sheet is being dismissed?

Reply:
Updated the README to clarify